packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -41,9 +50,6 @@ export const styles = (theme: ThemeType): JssStyles => ({
   },
   contentGivenImage: {
     marginTop: 185,
-    [theme.breakpoints.down('sm')]: {
-      marginTop: 130,
-    },
```

Comment:
Removing this fixes this problem on tablet-sized screens:
![Screenshot 2022-11-22 at 16 01 25](https://user-images.githubusercontent.com/5075628/203362217-950887a9-4ccf-46f8-b4a3-2f968e6314a3.png)

It's possible that it's needed for some other reason that I didn't notice though, so maybe there's a better fix here(?)